Config2016/src/Config.java

```diff
@@ -202,13 +202,37 @@ private void initConfig()
 				addToConstants("FLYWHEEL_MOTOR_2", 11);
 				addToConstants("FLYWHEEL_MOTOR_2_REVERSE", true);
 				addToConstants("FLYWHEEL_MOTOR_2_PDP_CHANNEL", 5);
-				addToConstants("FLYWHEEL_MOTOR_2_MAX_CURRENT", 1000.0);// TODO:
+				addToConstants("FLYWHEEL_MOTOR_2_MAX_CURRENT", 1000.0); // TODO:
 																		// Check
 																		// what
 																		// this
 																		// value
 																		// should
 																		// be.
+				
+				/*
+				 * Turret
+				 */
+				addToConstants("TURRET_MOTOR", 0);
+				addToConstants("TURRET_MOTOR_REVERSE", false);
+				addToConstants("TURRET_MOTOR_PDP_CHANNEL", 0);
+				addToConstants("TURRET_MOTOR_MAX_CURRENT", 1000.0); // TODO: Check the stall current
```

Comment:
If we're not sure what is the current limit - let's at least not merge this without setting this to a realistic number. 10 amps?